### DISCUSSION TITLE
Standalone esp8266

### DISCUSSION THREAD

**Discussion by duelago** [SwiCago/HeatPump#222] [Ideas]
EDIT: Just found "fancy web" Will test this :) 

Is there "standalone" Arduino code out there using this library? I.e I want to control the heatpump via the esp8266, but don't rely on mqtt and home assistant. I just want to set a daily timer, set the temperature and be able to switch on and off. The reason is that I want to do some heating in my summer house during the winter, but I don't want to setup a complete system with a broker and HA for just the pump.

**Comment by Injineers** [comment]
know this is old, but I switched my d1 mini clones over to Geoffs esphome version instead of the mitsubishi2mqtt
https://github.com/geoffdavis/esphome-mitsubishiheatpump

works well, but missing some of the features that the mitsu2mqtt has

Ie fan speed & the vane controls are wonky at the moment (im still working on implementing some things, but there are a bunch of post on configuring the vane, as well as using a external sensor (which is why I switched bc. my unit is above my heated 3d printer enclosure & that heat tends to trick the unit into thinking its hotter in the room then it actually is (IE over by my desk its like 5-7 * colder)
